src/progly/query.cc

```diff
@@ -418,7 +418,7 @@ void worker()
     uint64_t nrows_server_processed = 0;
     uint64_t eval2_start = getns();
 
-    if (query == "flatbuf") {
+    if (query == "flatbuf" || query == "arrow") {
```

Comment:
Is this check necessary?  The default is now flatbuf so that neither flatbuf nor arrow should need to be specified now.

Reply:
Done